src/zampy/datasets/converter.py

```diff
@@ -0,0 +1,74 @@
+"""Dataset formatter for different conventions."""
+import json
+from pathlib import Path
+import xarray as xr
+from pint import UnitRegistry
+
+
+CONVENTIONS = {
+    "ALMA": "src/zampy/conventions/ALMA.json",
+}
+
+
+def check_convention(convention):
+    """Check if the given convention is supported."""
+    if convention.upper() not in CONVENTIONS:
+        raise ValueError(
+            f"The '{convention}' convention is not supported.\n"
+            "Please check the available conventions in the `conventions` "
+            "directory and choose one from there."
+        )
+    else:
+        print(f"Start converting data to follow the '{convention}' convention.")
+
+
+def convert(dataset: xr.Dataset, fname: str, output_path: Path, convention: str):
```

Comment:
```suggestion
def convert(dataset: xr.Dataset, fname: str, output_path: Path, convention: Union[str, Path]):
```
Let's also allow for a user-defined convention file.